conf/messages

```diff
@@ -37,11 +37,28 @@ hts.create.account.p2=Ask customer if they understand - and accept the terms and
 hts.account.exists.title=Customer already has an account
 hts.account.exists.h1=Customer already has an account
 
-hts.account.created.title=Customer''s account has been created
-hts.account.created.h1=Customer''s account has been created
-hts.account.created.p1=You need to tell the customer, we''ll post them a:
-hts.account.created.ul1-li1=Welcome pack
-hts.account.created.ul1-li2=password
+hts.account-created.h1=Help to Save account created
+hts.account-created.customer-account-number.title=Customer''s account number
+hts.account-created.tell-customer.heading=Tell the customer:
+hts.account.created.tell-customer.ul1-li1=their account lasts for 4 years and this time starts now
+hts.account.created.tell-customer.ul1-li2=they can save from £1 to £50 each calendar month
+hts.account.created.tell-customer.ul1-li3=they have until the last day of the month to pay in each month
+hts.account.created.tell-customer.ul1-li4=they do not have to pay in each month but cannot make up for months they’ve missed
+hts.account-created.tell-customer.what-happens-next.header=What happens next
+hts.account-created.tell-customer.what-happens-next.title=We’'ll post the customer a:
+hts.account.created.tell-customer.what-happens-next.ul1-li1=Welcome pack
+hts.account.created.tell-customer.what-happens-next.ul1-li2=password
+hts.account.created.tell-customer.start-saving=The customer can start saving now through a bank transfer or standing order. Or they will need to wait until they get their password and Welcome pack in the post.
+
+hts.account-created.tell-customer.bank-transfer-details.header=Bank transfer and standing order details
+hts.account-created.tell-customer.bank-transfer-details.title=Details for the customer's bank:
+hts.account-created.tell-customer.bank-transfer-details.sortcode.text=Sort code:
+hts.account-created.tell-customer.bank-transfer-details.sortcode.value=60 89 77
+hts.account-created.tell-customer.bank-transfer-details.accountNumber.text=account number:
+hts.account-created.tell-customer.bank-transfer-details.accountNumber.value=10028471
+hts.account-created.tell-customer.bank-transfer-details.payment-reference-number.text=payment reference number
+hts.account-created.tell-customer.bank-transfer-details.payment-reference-number.value=[1234567890123]
```

Comment:
@AndyHWChung could you please let us know the correct `payment-reference-number` here ? and as discussed earlier these details are of NSI and should be static 
thanks

Reply:
Yes, those are the right details. However, I think it'd be better if we read those values from config, with fake details as default and having the real details only in app-config-prod/app-config-base. If NS&I are a bit funny about the word NS&I appearing in our public github repos I'm sure they'd be funny about their bank details in our public github repos.

Reply:
Are you sure the `payment-reference-number` is right ?

Reply:
i've added them to the app-config-base 
please merge this PR as well : https://github.com/hmrc/app-config-base/pull/1268